packages/hint-font-relative-units/README.md

```diff
@@ -0,0 +1,66 @@
+# font-relative-units (`@hint/hint-font-relative-units`)
+
+Ensure font styles use relative units instead of pixels.
+
+## Why is this important?
+
+If website stylesheets define certain styles in pixels, they cannot be easily overridden by custom stylesheets and browser plugins. Some people benefit from altering font size, line height, and letter spacing to make text easier to read.
+
+## What does the hint check?
+
+This hint examines `css` and `scss` files and checks that the following properties are not defined in pixels:
```

Comment:
I believe `less` files will be checked as well. @antross can you confirm?

Reply:
Yes, as will `sass` files (in addition to `scss`) and potentially more in the future as we add new parsers (e.g. CSS-in-JS). Plus inline style blocks in `html` files are also analyzed. 

Perhaps sticking with just "`css` files" is enough to get the point across. Or we could try something more generic like "files containing style information".